README.md

```diff
@@ -1,20 +1,18 @@
+Ghidra is a software reverse engineering (SRE) framework created and maintained by the [National Security Agency][nsa] Research Directorate. This framework includes a suite of full-featured, high-end software analysis tools that enable users to analyze compiled code on a variety of platforms including Windows, Mac OS, and Linux. Capabilities include disassembly, assembly, decompilation, graphing, and scripting, along with hundreds of other features. Ghidra supports a wide variety of process instruction sets and executable formats and can be run in both user-interactive and automated modes. Users can also develop their own Ghidra plug-in components and/or scripts using Java or Python.
 
-In support of NSA's Cybersecurity mission, Ghidra was built to solve scaling and teaming problems on complex SRE efforts, and to provide a customizable and extensible SRE research platform. NSA has applied Ghidra SRE capabilities to a variety of problems that involve analyzing malicious code and generating deep insights for SRE analysts who seek a better understanding of potential vulnerabilities in networks and systems.
+In support of the NSA's Cybersecurity mission, Ghidra was built to allow scaling and collaboration on complex SRE efforts, and to provide a customizable and extensible SRE research platform. The NSA has applied Ghidra SRE to a variety of problems that involve analyzing malicious code and generating deep insights for SRE analysts who seek a better understanding of potential vulnerabilities in networks and systems.
 
-This repository is a placeholder for the full open source release.
-Be assured efforts are under way to make the software available here.
+This repository is a placeholder for the full open-source release.
 In the meantime, enjoy using Ghidra on your SRE efforts, developing your
 own scripts and plugins, and perusing the over-one-million-lines of Java and
 Sleigh code released within the initial public release.
 The release can be downloaded from our [project homepage][project].
-Please consider taking a look at our [contributor guide][contrib] to see how
-you can participate in this open source project when it becomes available.
+Please take a look at our [contributor guide][contrib] to learn how
+you can participate in this open-source project when the code becomes available.
```

Comment:
“Open-source” is also another commonly repeated grammatical error. Both [gnu.org](https://www.gnu.org/philosophy/open-source-misses-the-point.en.html) and the [Linux Foundation](
https://www.linuxfoundation.org/projects/) refer to open source **without** a hyphen.

Reply:
If you can cite a source that considers the proper construction of a compound adjective a grammatical error, do share! "Open source project" can means "source project that is open". With the dash, there's no ambiguity. Can you tell the difference between "the three-letter abbreviation" and "the three letter abbreviation"?

One advantage of learning English as a fifth language is that ~~you~~ one can identify its grammatical inconsistencies instead of invoking teleological arguments.

Reply:
The hyphen is used to clarify ambiguity.

But “open source” and “open-source” have the same meaning; therefore, because there is no ambiguity, using it in this case is needless, and so should not be used. (Per the [University of Sussex](http://www.sussex.ac.uk/informatics/punctuation/hyphenanddash/hyphen).